project/Dependencies.scala

```diff
@@ -142,7 +149,7 @@ object Dependencies {
 
   val clusterMetrics = l ++= Seq(Provided.sigarLoader, Test.slf4jJul, Test.slf4jLog4j, Test.logback, Test.mockito)
 
-  val distributedData = l ++= Seq(Test.junit, Test.scalatest.value)
+  val distributedData = l ++= Seq(lmdb, lmdbOsx, lmdbLinux, lmdbWin, Test.junit, Test.scalatest.value)
```

Comment:
Well, they have to be resolved and downloaded for everyone, even if they do not use durable at all, so more a tidyness question than an actual problem (I think at least), just like that we do not pull in LevelDB for everyone from persistence.